packages/flutter/test/material/page_test.dart

```diff
@@ -297,7 +297,7 @@ void main() {
       ),
     );
     await tester.tap(find.text('PUSH'));
-    expect(await tester.pumpAndSettle(const Duration(minutes: 1)), 3);
+    expect(await tester.pumpAndSettle(const Duration(minutes: 1)), 2);
```

Comment:
These two lines were changed from 2 to 3 a while ago when hover was introduced. https://github.com/flutter/flutter/pull/31438/files#r283037816
This PR happens to change them back to 2, which is good.